# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1472091340 14400
#      Wed Aug 24 22:15:40 2016 -0400
# Node ID 805442f0674b7f092a6825c698a4b0d82b03f4a5
# Parent  9e3459ecbeb72cc3771fe6024d57e94123d04b62
* test.m: Fix doubled "!!!!!" marker in failure messages.

diff --git a/scripts/testfun/test.m b/scripts/testfun/test.m
--- a/scripts/testfun/test.m
+++ b/scripts/testfun/test.m
@@ -606,27 +606,27 @@ function [__n, __nmax, __nxfail, __nskip
         catch
           if (isempty (lasterr ()))
             error ("test: empty error text, probably Ctrl-C --- aborting");
           else
             __success = false;
             if (__isxtest)
               __xfail += 1;
               if (isempty (__bug_id))
-                __msg = [__signal_fail "known failure\n"];
+                __msg = "known failure";
               else
                 if (all (isdigit (__bug_id)))
                   __bug_id = ["http://octave.org/testfailure/?" __bug_id];
                 endif
-                __msg = ["known bug: " __bug_id "\n"];
+                __msg = ["known bug: " __bug_id];
               endif
             else
-              __msg = "test failed\n";
+              __msg = "test failed";
             endif
-            __msg = [__signal_fail __msg lasterr()];
+            __msg = [__signal_fail __msg "\n" lasterr()];
           endif
         end_try_catch
         clear __test__;
       endif
 
       ## All done.  Remember if we were successful and print any messages.
       if (! isempty (__msg) && (__verbose >= 0 || __logfile))
         ## Make sure the user knows what caused the error.
